### DISCUSSION TITLE
Subject-level bootstrap in rfsrc

### DISCUSSION THREAD

**Discussion by Vaggelis-Panagiotopoulos** [kogalur/randomForestSRC#380] [Q&A]
I have a repeated measurements data set, hence, as pointed out in "An introspective comparison of random forest-based classifiers for the analysis of cluster-correlated data by way of RF++" paper  by Karpievitch YV, Hill EG, Leclerc AP, I would like to build a RF where each tree is built upon a bootstrap replicate taken at subject-level.

I constructed a function creating an appropriate matrix to be used as 'samp' argument in rfsrc() but my implementation is not computationally efficient. Does the randomForestSRC package provide any way to do this in a computationally cheap manner?

*Note that I don't want to perform a survival analysis, since my data set does not contain an event variable.

**Comment by min858** [comment]
Hello, may I ask how you construct the 'samp' argument in rfsrc()?
